app/models/worker/deposit_coin_address.rb

```diff
@@ -8,11 +8,13 @@ def process(payload, metadata, delivery_info)
       return if payment_address.address.present?
 
       currency = payload[:currency]
+      c = Currency.find_by_code(currency.to_s)
+
       if currency == 'eth'
         address  = CoinRPC[currency].personal_newAccount("")
-        open('http://your_eth_server_ip/cgi-bin/restart.cgi')
+        open(c.rpc + '/cgi-bin/restart.cgi')
```

Comment:
@muhammednagy Why there is a need to call this URL? Is it necessary to reboot after we create a new account using `CoinRPC['eth'].personal_newAccount("")` ?

Reply:
i will remove this line 
it was necessary 
it isn't now